package.json

```diff
@@ -24,13 +24,16 @@
     "history": "5",
     "i18next": "^21.4.2",
     "i18next-browser-languagedetector": "^6.1.2",
+    "prop-types": "^15.0.0-0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-i18next": "^11.14.0",
     "react-redux": "^7.2.0",
     "react-router-dom": "6",
     "react-scripts": "4.0.3",
+    "react-styled-flexboxgrid": "^3.2.1",
     "react-test-renderer": "^17.0.2",
+    "styled-components": "2",
```

Comment:
I'm going to approve the PR but I'm curious if the `styled-components` package is needed (not a big deal but I believe that emotion already added it).

Reply:
Nevermind. I'll have to add it for the GlobalStyles.

Reply:
haha yes, and also for the installation, they mention to add them XD
![image](https://user-images.githubusercontent.com/29077688/143504630-664bc793-2b72-494b-85ee-57f00f25d276.png)